_dev/css/components/quickview.scss

```diff
@@ -37,7 +37,7 @@
     display: flex;
     min-height: 21.88rem;
 
-    .product-images > li.thumb-container > .thumb {
+    .product-images li.thumb-container > .thumb {
```

Comment:
Why do you need to change this? You changed the specificity of this selector. What was the reason?

Reply:
A typo, changed. :-)